tests/libcxx-backports/MinimalElementType.h

```diff
@@ -25,20 +25,22 @@ struct MinimalElementType {
 template<class T, size_t N>
 struct ElementPool {
   constexpr ElementPool() {
-    ptr_ = std::allocator<T>().allocate(N);
+    nc_ptr_ = std::allocator<std::remove_const_t<T>>().allocate(N);
```

Comment:
I don't get this change either

Reply:
libc++ accepts `std::allocator<const int>` libstdc++ says that is an invalid specialization of allocator.